### ISSUE TITLE
addons.mozilla.org is being blocked by Russian authorities

### ISSUE THREAD

**Issue by roman90sv** [mozilla/addons#714]
"Roskomnadzor" _(Federal Service for Supervision of Communications, Information Technology and Mass Media)_ is now blocking a wide range of IP addresses, including addresses that belong to mozilla:
34.192.0.0/10
34.192.0.0/10
54.64.0.0/13
So, the website is not always available to users in Russia.

UPD:
+accounts.firefox.com by 35.160.0.0/13 & 54.144.0.0/12

I know how to solve this problem - VPN. But I wanted to notify mozilla team about what happened.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/AMOENG-35)

**Comment by EnTeQuAk** [comment]
Thanks for the notification! We are aware of that but in the short-term there doesn't seem to be much wen can do. This appears to be related to the Telegram block.

**Comment by roman90sv** [comment]
Yes, it's consequences of attempts by the censor to block telegram. Even VPS which is used as a proxy server are blocked.
It's a shame for the country...

**Comment by mniip** [comment]
Someone (preferably a russian individual) could file a report at https://rkn.gov.ru/treatments/ask-question/ . No guarantees about a swift reply though...

**Comment by jvillalobos** [comment]
@roman90sv Can you check again? We got some reports that AMO isn't being blocked in Russia currently.

**Comment by roman90sv** [comment]
**@jvillalobos** I don't think you understand the gravity of this situation...

![00](https://user-images.githubusercontent.com/35082918/40136648-09340c9a-5951-11e8-9628-bc149a547005.png)

![770](https://user-images.githubusercontent.com/35082918/40136968-f89dab42-5951-11e8-83b9-0db1e4229ae2.png)

Sources
@krnshowtimebot
@rkn_block_check
https://t.me/rkn_digest
https://t.me/rknshowtime

**Comment by jvillalobos** [comment]
@roman90sv we asked people in Russia to try connecting to AMO and they didn't report any problems. I'm asking to try to figure out if this is a regional problem, a temporary one, or something else.

**Comment by roman90sv** [comment]
The website is available now (for me). But it's not sure that the website will be available tomorrow.
Some users complained about the problems with synchronization.

This is a regional problem (varies between ISPs)+temporary problem (blocking/unblocking stuff randomly evryday)+something else.
:)

**Comment by roman90sv** [comment]
**@jvillalobos**
Domain is not unavailable again.
https://drive.google.com/open?id=1Bi4MyLb4RnhqnSk_K521I9lEqb_HiB2f

**Comment by sigprof** [comment]
It seems that the set of IP addresses for addons.mozilla.org changes regularly, therefore the problem can appear and disappear again.

Yesterday addons.mozilla.org was resolving into 34.209.9.196, 35.166.74.225, 35.167.92.222; all those addresses were blocked in Russia (34.192.0.0/10 and 35.160.0.0/13 network ranges are blocked), therefore https://addons.mozilla.org/ was completely unreachable.

Today addons.mozilla.org resolves into 35.166.74.225, 35.167.92.222, 52.89.239.72. Two of these addresses (35.166.74.225 and 35.167.92.222) are still blocked in Russia, but the 52.89.239.72 address is reachable, therefore https://addons.mozilla.org/ is available, but with long delays caused by timeouts for unreachable addresses (my ISP just drops packets to blocked addresses).

**Comment by roman90sv** [comment]
**+accounts.firefox.com by 35.160.0.0/13 & 54.144.0.0/12**
![4490](https://user-images.githubusercontent.com/35082918/40980208-8b5815d8-68e0-11e8-8de5-c2c6431b7a07.PNG)

**Comment by grahamperrin** [comment]
Might IPFS lend itself to issues such as this in the future?

**Comment by rugk** [comment]
Also FYI https://isitblockedinrussia.com/?host=addons.mozilla.org may help.
